OsmAnd/src/net/osmand/plus/myplaces/tracks/dialogs/SplitSegmentsAdapter.java

```diff
@@ -104,6 +115,26 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
 				if (trackGroup != null && (trackGroup.isSplitDistance() || slopeType != null)) {
 					if (slopeType != null) {
 						overviewImageView.setImageDrawable(getSlopeDrawable(slopeType, nightMode));
+
+						int color = app.getSettings().getApplicationMode().getProfileColor(nightMode);
+						Drawable background = UiUtilities.getColoredSelectableDrawable(app, color, 0.3f);
+						AndroidUtils.setBackground(headerButton, background);
+
+						headerButton.setOnClickListener(v -> {
+							SelectedGpxFile selectedGpxFile;
+							selectedGpxFile = app.getSelectedGpxHelper().getSelectedFileByPath(currentGpxDisplayItem.group.getGpxFile().getPath());
+
+							if (selectedGpxFile != null) {
+								listener.onDismiss();
```

Comment:
Strange listener method, better move whole block with opening context menu to fragment via listener